src/software/simulated_tests/validation_functions/ball_velocity_validation.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+#include "software/simulated_tests/validation/validation_function.h"
+#include "software/world/world.h"
+
+void ballExceedsVelocity(std::shared_ptr<World> world_ptr,
```

Comment:
please add a javadoc comment for this function

Also to help be clear we're testing against the max allowed velocity by the rules (not a velocity provided as a parameter), perhaps a more descriptive name would be `ballExceedsMaxVelocity`